test/Dialect/SPIRV/Serialization/function_call.mlir

```diff
@@ -0,0 +1,68 @@
+// RUN: mlir-translate -serialize-spirv %s | mlir-translate -deserialize-spirv | FileCheck %s
+
+spv.module "Logical" "GLSL450" {
+  spv.globalVariable @var1 : !spv.ptr<!spv.array<4xf32>, Input>
+  func @fmain() -> () {
+    %0 = spv.constant 16 : i32
+    %1 = spv._address_of @var1 : !spv.ptr<!spv.array<4xf32>, Input>
+    // CHECK: {{%.*}} = spv.FunctionCall @f_0({{%.*}}) : (i32) -> i32
+    %3 = spv.FunctionCall @f_0(%0) : (i32) -> i32
+    // CHECK: spv.FunctionCall @f_1({{%.*}}, {{%.*}}) : (i32, !spv.ptr<!spv.array<4 x f32>, Input>) -> ()
+    spv.FunctionCall @f_1(%3, %1) : (i32, !spv.ptr<!spv.array<4xf32>, Input>) ->  ()
+    // CHECK: spv.FunctionCall @f_2() : () -> ()
+    spv.FunctionCall @f_2() : () -> ()
+    // CHECK: spv.FunctionCall @f_3({{%.*}}) : (!spv.ptr<!spv.array<4 x f32>, Input>) -> !spv.ptr<!spv.array<4 x f32>, Input>
+    %4 = spv.FunctionCall @f_3(%1) : (!spv.ptr<!spv.array<4xf32>, Input>) -> !spv.ptr<!spv.array<4xf32>, Input>
+    // CHECK: spv.FunctionCall @f_3({{%.*}}) : (!spv.ptr<!spv.array<4 x f32>, Input>) -> !spv.ptr<!spv.array<4 x f32>, Input>
```

Comment:
Do we need this case? It doesn't cover something new compared to the above one.

Reply:
I was thinking to check, that we can pass a result value from FunctionCall as operand for the other FunctionCall.

Reply:
The result of a function call is a normal SSA value, so nothing different from other SSA values. I don't think we need to check it. :)

Reply:
@antiagainst it's just check fro my first invalid implementation, I was missing to put and id of OpFunctionCall into id valueMap for serializer. I'll delete, thanks!

Reply:
Oh, I see. In that case, you can `spv.Return %3 : i32` or something. No need to create another `spv.FunctionCall` just to use the result of a `spv.FunctionCall`. :)

Reply:
@antiagainst thanks, got it!